routes/socket/user-events.js

```diff
@@ -2140,24 +2140,54 @@ module.exports.handleAddNewGameChat = (socket, passport, data, game, modUserName
 	const pingMods = /^@(mod|moderator|editor|aem|mods) (.*)$/i.exec(chat);
 
 	if (pingMods && player) {
-		if (!game.lastModPing || Date.now() > game.lastModPing + 180000) {
-			game.lastModPing = Date.now();
-			sendInProgressGameUpdate(game, false);
-			makeReport(
-				{
-					player: passport.user,
-					situation: `"${pingMods[2]}".`,
-					election: game.general.electionCount,
-					title: game.general.name,
-					uid: game.general.uid,
-					gameType: game.general.casualGame ? 'Casual' : game.general.practiceGame ? 'Practice' : 'Ranked'
-				},
-				game,
-				'ping'
+		Account.find({ staffRole: { $exists: true } }).then(accounts => {
```

Comment:
Not in scope for this PR, but I'd like to get this whole thing extracted back out to its own function. I think the workings of one exist with `getPowerFromRole` or something, but idt it gets used.